R-package/R/lgb.prepare2.R

```diff
@@ -37,6 +38,7 @@
 #' # $ Petal.Length: num  1.4 1.4 1.3 1.5 1.4 1.7 1.4 1.5 1.4 1.5 ...
 #' # $ Petal.Width : num  0.2 0.2 0.2 0.2 0.2 0.4 0.3 0.2 0.2 0.1 ...
 #' # $ Species     : int  1 1 1 1 1 1 1 1 1 1 ...
+#' }
```

Comment:
@jameslamb I added this because after calling `lgb.unloader()` all examples crash. Please take a look at our previous conversation with @Laurae2 : 3 comments starting from https://github.com/microsoft/LightGBM/pull/2176#issuecomment-493138605.
Please help to choose the appropriate guard which will prevent running these pieces of code during documentation (examples part) generation process.